message_ix_models/tools/exo_data.py

```diff
@@ -43,9 +44,12 @@ class ExoDataSource(ABC):
     key: Optional[Key] = None
 
     #: Name for the returned :class:`.Key`/:class:`.Quantity`. Optional. See
-    # :meth:`get_keys`.
+    #: :meth:`get_keys`.
     name: str = ""
 
+    #: Primary measure.
+    measure: str = ""
```

Comment:
```suggestion
    measure = ""
```
Shouldn't this be enough for any type checker to infer the type of `measure`?

Reply:
Hm, true. Thanks!